md/access-control-api.md

```diff
@@ -0,0 +1,56 @@
+# Access control API
+
+::: info
+**Note:** This API is only available in Efficiency and Performance edition.
+:::
+
+## BDM
+
+#### Description
+
+Access control is available to protect your BDM by tenant. You can use this API to get access control status (lastUpdateBy, lastUpdateDate...).
+
+#### Methods
+ * GET - Read access control status.
+ * DELETE - Remove access control
+ 
+::: info
+**Note:** A servlet to import access control file exist. You can use it with this url `services/bdmAccessControl/install?bdmAccessControlUpload=accessControlFileToUpload.xml`.
```

Comment:
exists
the upload needs to be performed before (POST portal/bdmAccessControlUpload)